servers/camera_server.h

```diff
@@ -104,8 +104,6 @@ class CameraServer : public Object {
 	int get_feed_count();
 	Array get_feeds();
 
-	RID feed_texture(int p_id, FeedImage p_texture);
```

Comment:
This one should be checked by @BastiaanOlij, it's `public` so maybe it's used from GDNative implementations.

Reply:
Yup, these functions are added with the idea that you extend these classes to implement the camera server. Just because they aren't used in the core does not mean they are useless.

Reply:
I'll add a comment then.